.github/CONTRIBUTING.md

```diff
@@ -1,47 +1,22 @@
 # Contributing to PowerShell
 
 We welcome and appreciate contributions from the community.
-There are many ways to become involved with PowerShell:
-including filing issues,
-joining in design conversations,
-writing and improving documentation,
-and contributing to the code.
-Please read the rest of this document to ensure a smooth contribution process.
-
-## Intro to Git and GitHub
-
-* Make sure you have a [GitHub account](https://github.com/signup/free).
-* Learning Git:
-  * GitHub Help: [Good Resources for Learning Git and GitHub][good-git-resources]
-  * [Git Basics](../docs/git/basics.md): install and getting started
-* [GitHub Flow Guide](https://guides.github.com/introduction/flow/):
-  step-by-step instructions of GitHub Flow
-
-## Quick Start Checklist
-
-* Review the [Contributor License Agreement][CLA] requirement.
-* Get familiar with the [PowerShell repository](../docs/git).
-
-## Contributing to Issues
+There are many ways to become involved with PowerShell including:
+- filing and participating in issues
+- joining in design conversations
+- writing and improving documentation
+- contributing to the code
```

Comment:
Can we add links to docs for every case? 
I see we start with "documentation" but should with "issues".
And I don't see "design conversations" part. (It is optional request.)